e2e-tests/seedTimestream.ts

```diff
@@ -0,0 +1,37 @@
+import { WriteRecordsCommand } from '@aws-sdk/client-timestream-write'
+import { usageToRecord } from '../lambda/usageToRecord.js'
+import {
+	type _Record,
+	type TimestreamWriteClient,
+} from '@aws-sdk/client-timestream-write'
+
+export const seedTimestream =
+	(tsw: TimestreamWriteClient) =>
+	async (
+		timestamps: Array<Date>,
+		values: Array<number>,
+		iccid: string,
+		dbName: string,
+		tableName: string,
+	): Promise<void> => {
+		const records = timestamps.map((timestamp, index) => {
+			const record = usageToRecord({
+				iccid,
+				diff: values[index] ?? 0,
+				currentTime: timestamp,
+			})
+			if ('record' in record) {
+				return record.record
+			}
+			return null
+		})
+		const isNotNull = (value: _Record | null) => value != null
+		const filteredRec = records.filter(isNotNull)
```

Comment:
**nitpick**: move to outside of seedTimestream, does not need to be redefined for every call.